packages/opentelemetry-node/src/instrumentation/PluginLoader.ts

```diff
@@ -187,9 +187,13 @@ export class PluginLoader {
           this._plugins.push(plugin);
           // Enable each supported plugin.
           return plugin.enable(exports, this.provider, this.logger, config);
-        } catch (e) {
-          this.logger.error(
-            `PluginLoader#load: could not load plugin ${modulePath} of module ${name}. Error: ${e.message}`
+        } catch (err) {
+          const logLevel =
+            err instanceof Error && err.message.match(/Cannot find module/)
```

Comment:
```suggestion
            err instanceof Error && err.message.match(/Cannot find module/ || /with the versions/)
```

Pardon me for extending the scope, but would it be possible to reduce to warn for when the app's module is not a traceable version?

```json
{
  "level": "error",
  "severity": "ERROR",
  "eventTime": 1600053551.793,
  "serviceContext": {
    "service": "foo"
  },
  "message": "PluginLoader#load: Plugin redis only supports module redis with the versions: ^2.6.0"
}
```

I would not consider these an error per se, though good to know about.

Reply:
We can address this in a seperate PR, also not even sure this is the right place for my suggestion.

Reply:
https://github.com/open-telemetry/opentelemetry-js/pull/1532 created